hushline/routes.py

```diff
@@ -235,7 +239,9 @@ def register() -> Response | str:
             flash("👍 Registration successful! Please log in.", "success")
             return redirect(url_for("login"))
 
-        return render_template("register.html", form=form, require_invite_code=require_invite_code)
+        return (
+            render_template("register.html", form=form, require_invite_code=require_invite_code),
```

Comment:
1-tuple

Reply:
in this case its happening from the code formatters.